package.json

```diff
@@ -32,45 +32,45 @@
     "@angular/platform-browser": "^4.4.6",
     "@angular/platform-browser-dynamic": "^4.4.6",
     "@angular/router": "^4.4.6",
-    "@types/geojson": "^1.0.3",
-    "bootstrap-sass": "^3.3.7",
+    "@types/geojson": "^7946.0.5",
+    "bootstrap-sass": "^3.4.0",
     "climate-change-components": "^0.2.2",
-    "core-js": "^2.5.1",
-    "d3": "^4.10.0",
-    "d3-tip": "^0.7.1",
+    "core-js": "^2.6.4",
+    "d3": "^4.13.0",
+    "d3-tip": "^0.9.1",
     "enhanced-resolve": "^3.4.1",
-    "file-saver": "^1.3.3",
-    "jquery": "^3.2.1",
+    "file-saver": "^1.3.8",
+    "jquery": "^3.3.1",
     "lodash.clonedeep": "^4.5.0",
-    "moment": "^2.18.1",
+    "moment": "^2.24.0",
     "ng2-nouislider": "^1.6.2",
     "ngx-bootstrap": "^1.9.3",
-    "ngx-clipboard": "^8.0.4",
+    "ngx-clipboard": "^8.1.4",
     "nouislider": "^10.1.0",
     "rxjs": "^5.5.2",
-    "save-svg-as-png": "^1.2.0",
-    "zone.js": "0.8.17"
+    "save-svg-as-png": "^1.4.9",
+    "zone.js": "0.8.29"
   },
   "devDependencies": {
     "@angular/cli": "1.3.2",
     "@angular/compiler-cli": "^4.4.6",
     "@angular/language-service": "^4.4.6",
-    "@types/jasmine": "2.5.54",
-    "@types/jquery": "^3.2.12",
-    "@types/node": "~8.0.26",
-    "codelyzer": "~3.1.2",
-    "jasmine-core": "~2.8.0",
+    "@types/jasmine": "2.8.9",
+    "@types/jquery": "^3.3.29",
+    "@types/node": "~11.9.3",
+    "codelyzer": "~3.2.2",
+    "jasmine-core": "~2.9.1",
     "jasmine-spec-reporter": "~4.2.1",
     "karma": "~1.7.1",
     "karma-chrome-launcher": "~2.2.0",
     "karma-cli": "~1.0.1",
-    "karma-coverage-istanbul-reporter": "^1.3.0",
-    "karma-jasmine": "~1.1.0",
+    "karma-coverage-istanbul-reporter": "^1.4.3",
+    "karma-jasmine": "~1.1.2",
     "karma-jasmine-html-reporter": "^0.2.2",
-    "protractor": "~5.1.2",
+    "protractor": "~5.4.2",
     "ts-node": "~3.3.0",
-    "tslint": "~5.7.0",
+    "tslint": "~5.12.1",
```

Comment:
By reverting the update to `tslint` I was able to get linting to pass again. The `e2e/` directory is used for end-to-end tests, which this project doesn't have. It looks like the support for `e2e` was somewhat removed, but not enough to make the latest version of `tslint` happy. I briefly tried removing further references to `e2e` but I wasn't able to get the new `tslint` to stop complaining about the missing file.

Reply:
:+1: Pushed up a commit to revert the tslint upgrade. Thanks for the help with this!